src/handlers/wildcard/unassign.ts

```diff
@@ -49,7 +51,14 @@ const checkBountyToUnassign = async (issue: Issue): Promise<boolean> => {
   const curTimestamp = new Date().getTime();
   const lastActivity = await lastActivityTime(issue, comments);
   const passedDuration = curTimestamp - lastActivity.getTime();
+  const pullRequest = await getOpenedPullRequestsForAnIssue(issue.number, issue.assignee);
 
+  const reviews = await getPullRequestReviews(context, pullRequest[0].id, 30, 1);
+
+  const pendingReviews = reviews.filter((review) => review.state === "PENDING");
```

Comment:
[pull request reviews](https://docs.github.com/en/rest/pulls/reviews?apiVersion=2022-11-28#list-reviews-for-a-pull-request) api will only return the reviews which state can be `COMMENT`, `CHANGES_REQUESTED`, or `APPROVED`. I think you need to combine this with [review requests](https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#about-review-requests) to see who was requested but didn't leave a review yet, but still that can't be accurate because if the reviewer leaves a review and creator fixes things and requests a review again so you shouldn't count previous reviews.

Reply:
How would you combine them?

Reply:
@whilefoo how would you combine them

Reply:
like I typed - you would use [review requests](https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#about-review-requests) to see who was requested and use [pull request reviews](https://docs.github.com/en/rest/pulls/reviews?apiVersion=2022-11-28#list-reviews-for-a-pull-request) to see who left the review and who didn't.

however this is not accurate as I described above, so you should try to find another way

Reply:
> if for example 0xcodercrane left a review he would be removed from review requests right?

if that's true, that would be great. can you verify it?

Reply:
trying rn

Reply:
@whilefoo if i request changes it removes me from 
https://api.github.com/repos/EtherealGlow/ubiquibot/pulls/15/requested_reviewers

Reply:
https://github.com/EtherealGlow/ubiquibot/pull/15